extension/views/lexicon.html.ejs

```diff
@@ -260,6 +260,7 @@
             Example
             <p>Copy (copies whatever is selected)</p>
             <p>Copy link</p>
+            <p>Copy links</p>
```

Comment:
We don't need to tell users about this specific phrasing, it's just there so that if it happens to be transcribed as "copy links" it will still work. So this can be removed.